pkg/solana/relay.go

```diff
@@ -50,51 +45,61 @@ func (r *Relayer) Name() string {
 }
 
 // Start starts the relayer respecting the given context.
-func (r *Relayer) Start(context.Context) error {
+func (r *Relayer) Start(ctx context.Context) error {
 	// No subservices started on relay start, but when the first job is started
 	if r.chain == nil {
 		return errors.New("Solana unavailable")
 	}
-	return nil
+	return r.chain.Start(ctx)
 }
 
 // Close will close all open subservices
-func (r *Relayer) Close() error {
-	r.cancel()
-	return nil
-}
+func (r *Relayer) Close() error { return r.chain.Close() }
 
 func (r *Relayer) Ready() error {
 	return r.chain.Ready()
 }
 
-// Healthy only if all subservices are healthy
 func (r *Relayer) Healthy() error { return nil }
 
 func (r *Relayer) HealthReport() map[string]error {
-	return map[string]error{r.Name(): r.Healthy()}
+	hp := map[string]error{r.Name(): r.Healthy()}
+	services.CopyHealth(hp, r.chain.HealthReport())
+	return hp
+}
+
+func (r *Relayer) GetChainStatus(ctx context.Context) (relaytypes.ChainStatus, error) {
+	return r.chain.GetChainStatus(ctx)
+}
+
+func (r *Relayer) ListNodeStatuses(ctx context.Context, pageSize int32, pageToken string) (stats []relaytypes.NodeStatus, nextPageToken string, total int, err error) {
+	return r.chain.ListNodeStatuses(ctx, pageSize, pageToken)
+}
+
+func (r *Relayer) Transact(ctx context.Context, from, to string, amount *big.Int, balanceCheck bool) error {
+	return r.chain.Transact(ctx, from, to, amount, balanceCheck)
```

Comment:
breaking interface changes?

will probably want to get everything running between this PR and core PR before merging this one (trying to reduce the amount of time something would be broken)

Reply:
Yeah I wanted to get these clean and collapse them all quickly, but we're clogged up on tdh2 now unfortunately. I will wait to open this until it is ready.